events.html

```diff
@@ -32,14 +32,26 @@
       Web Mini-Conference | Important concepts a Web Developer should know
     </title>
     <link rel="stylesheet" href="./css/main.css" />
-    <link rel="stylesheet" href="./css/events.css" />
+		<link rel="stylesheet" href="./css/events.css" />
+		<link rel="stylesheet" href="./css/navbar.css">
```

Comment:
Why the extra indentation?

Reply:
There isn't but github shows it that way 
Ah, must be that my tab is set to 4 instead of 2 spaces 🤦